CHANGELOG.md

```diff
@@ -2,15 +2,47 @@
 
 ## Unreleased
 
-## 🛑 Breaking changes 
+## v0.7.0 Beta
+
+## 🛑 Breaking changes 🛑
+
+- Replaced `disabled` with `enabled` in `sending_queue` and `retry_on_failure` config options (#1433)
```

Comment:
This is not breaking change I think because the PR that added them was not released

Reply:
@bogdandrutu Should I move the line to another section or just remove it?

Reply:
Just remove it :)